metabase/metabase
# Show timeline events in charts

Epic: https://github.com/metabase/metabase/issues/20289
Original PR created by Anton: https://github.com/metabase/metabase/pull/20701

Changes to the original PR:
- Changed the way how timelines are loaded. Now they are loaded in parallel with other referenced entities, e.g. tables, databases, etc.
- Tooltip content and date formatting changes
- Bugfixes

How to test:
- Go to the root collection and create several events for years 2017-2019.
- Create and save a new question, Orders, group by Created date
- Make sure events are displayed in the query builder for this question
- It should be possible to hover event icons and see their details

<img width="1728" alt="Screenshot 2022-03-08 at 22 16 07" src="https://user-images.githubusercontent.com/8542534/157309327-afb5d56b-e5e8-48ed-8bda-a473afaf9006.png">
